### ISSUE TITLE
Text parameter order in multipart/form-data

### ISSUE THREAD

**Issue by mefernandez** [balderdashy/skipper#87]
I'm using Sails 0.11.0 with Skipper 0.5.5. I'm trying to POST a file upload, which looks like so:
```json
------WebKitFormBoundaryWlJMv7RcZg3QnzE4
Content-Disposition: form-data; name="fileName"

Aprender en el siglo XXI_statusred.gif
------WebKitFormBoundaryWlJMv7RcZg3QnzE4
Content-Disposition: form-data; name="type"

imageList
------WebKitFormBoundaryWlJMv7RcZg3QnzE4
Content-Disposition: form-data; name="radioId"

3
------WebKitFormBoundaryWlJMv7RcZg3QnzE4
Content-Disposition: form-data; name="fileIndex"

3
------WebKitFormBoundaryWlJMv7RcZg3QnzE4
Content-Disposition: form-data; name="file"; filename="status-red.gif"
Content-Type: image/gif


------WebKitFormBoundaryWlJMv7RcZg3QnzE4--
```

Everything works fine in localhost, but the same code with the same call is failing in the staging server. 

Env info:
- **localhost**:
  - Mac OSX 10.6
  - node v0.12.1
- **staging**:
  - Docker container, running the official node image
  - node v0.12.2

I'm guessing the problem is not in how the browser sends the request, but something in the server. I've used Chrome's Rest Console extension to replay the same request pointing to both envs, and the result is Ok for localhost and failure for staging.

**Comment by mefernandez** [comment]
I created a container running node 0.10.38 from [Docker Hub's official node image](https://github.com/joyent/docker-node/blob/428d5e69763aad1f2d8f17c883112850535e8290/0.10/Dockerfile), still the same result.

By the way, the connection to the server is **not** HTTPS (I've read some issues with HTTPS and field ordering).

**Comment by mefernandez** [comment]
I've finally made it work. I was accessing the Sails app directly on port 1337. I then switched to serving it on port 80 via nginx, and it started to work fine. So , I guess the hosting provider (1&1 in this case) might be doing something funny on non-standard ports that messes up the multipart, but that's just a theory anyway.

**Moral**: If this is happening to you, try non-HTTPS, try standard port 80, and try no to go through  things that might change the request like firewalls, proxies, etc.
